app/models/katello/glue/provider.rb

```diff
@@ -136,6 +136,7 @@ def import_products_from_cp
         end
         prod_content_importer.import
         self.index_subscriptions(self.organization)
+        prod_content_importer.content_url_updated
```

Comment:
it seems strange that a method called `import_products_from_cp` would return a list of content objects that only had their urls change. maybe it should return the whole Importer object and your action can be adjusted accordingly